spaces/S000044/properties/P000086.md

```diff
@@ -4,5 +4,4 @@ property: P000086
 value: false
 ---
 
-$1/3$ belongs to a single open set $(0,1)$, but
-$2/5$ belongs to two open sets $(0,1)$ and $(0,1/2)$.
+$1$ is a closed point, but $1/3$ is not.
```

Comment:
$1$ doesn't belong to the space, so this needs to be fixed. I'll address in #791